pkg/csi/plugins/nodeserver.go

```diff
@@ -167,6 +172,23 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 func (ns *nodeServer) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpublishVolumeRequest) (*csi.NodeUnpublishVolumeResponse, error) {
 	targetPath := req.GetTargetPath()
 
+	// check path existence
+	_, err := os.Stat(targetPath)
+	// No need to unmount non-existing targetPath
+	if os.IsNotExist(err) {
+		glog.V(4).Infof("NodeUnpublishVolume: targetPath %s does not exist", targetPath)
```

Comment:
I suggest changing logging info to `NodeUnpublishVolume: targetPath %s has been cleaned up, so it doesn't need to be unmounted.`

Reply:
just follow other success log such as "glog.V(4).Infof("Succeed in binding %s to %s", mountPath, targetPath)"